examples/java/README.md

```diff
@@ -0,0 +1,80 @@
+# Running the example
+Several functions have been added to this plugin to give an example of how you can add your own functions to a java plugin. Evaluate script supports script evaluation without input parameters and script evaluation 
+with input parameters if it is not an aggregation and if the script can be evaluated for one row of input parameters independently of the other rows of parameters. The scripts being evaluated however are JavaScript scripts 
+since there is no "eval" function in java.
+
+Synchronous/asynchronous:
```

Comment:
For me, this doesn't provide much useful info for the purpose of running an example plugin.